test/integration/master/api/providers_controller_test.rb

```diff
@@ -210,6 +210,7 @@ def setup
       assert_equal '', response.body
       assert provider.reload.scheduled_for_deletion?
       assert_equal Time.zone.now.to_s, provider.deleted_at.to_s
+      assert_equal Time.zone.now.to_s, provider.state_changed_at.to_s
```

Comment:
Maybe trying that thing of saving `Time.zone.now` to a variable before asserting to prevent infinitesimal differences messing up with the tests?

Reply:
There shouldn't be difference for the `Timecop.freeze`:
https://github.com/3scale/porta/blob/088a699ea7135ab2264ba04d256139b0f959cb44/test/integration/master/api/providers_controller_test.rb#L207-L214

I am not sure that stubbing the `Time.zone.now` for an integration test is a good idea 🤔 because it should be more agnostic to `how` it is done.

Reply:
Sorry, forget about it. I missed `Timecop.freeze` in GH crop of the code.